deploy/deployment.yaml

```diff
@@ -18,6 +18,11 @@ spec:
       containers:
         - name: aws-account-shredder
           image: quay.io/agautam/aws-account-shredder:latest
+          resources:
+            requestes:
+              memory: "1024Mi"
```

Comment:
Request for 1024Mi is too much for your service. I suggest starting from 100Mi
Btw, you also need to add this part to the template file

Reply:
changed to 100Mi, also added this to template.yaml